ydb/core/tx/schemeshard/schemeshard_import.cpp

```diff
@@ -28,6 +29,47 @@ namespace {
         return timestamp;
     }
 
+    void FillItemProgress(TSchemeShard* ss, const TImportInfo::TPtr importInfo, ui32 itemIdx,
+            Ydb::Import::ImportItemProgress& itemProgress) {
+
+        Y_ABORT_UNLESS(itemIdx < importInfo->Items.size());
+        const auto& item = importInfo->Items.at(itemIdx);
+
+        const auto opId = TOperationId(item.WaitTxId, FirstSubTxId);
+        if (item.WaitTxId != InvalidTxId && ss->TxInFlight.contains(opId)) {
+            const auto& txState = ss->TxInFlight.at(opId);
+            if (txState.TxType != TTxState::TxRestore) {
+                return;
+            }
+
+            itemProgress.set_parts_total(txState.Shards.size());
+            itemProgress.set_parts_completed(txState.Shards.size() - txState.ShardsInProgress.size());
+            *itemProgress.mutable_start_time() = SecondsToProtoTimeStamp(txState.StartTime.Seconds());
+        } else {
+            if (!ss->Tables.contains(item->DstPathId)) {
```

Comment:
`item.DstPathId`

`->` notation is used, when you accessing field through pointer